README.md

```diff
@@ -19,11 +19,12 @@ Each **tutorial** in ProtoSchool is made up of multiple **lessons**.
 
 First, check out this repository and run the dev server locally.
 
-```sh
+```console
```

Comment:
Why this change? `sh` colors the prompt for me, while `console` doesn't.

Reply:
yeah, I did half a change there. `sh` colors `>` because it thinks it is a greater than operator in a shell script. `console` makes more sense for blocks that you want to style as if they were commands entered at a prompt. Refardless, The readme looks better if I leave it as `sh` in this case,  so I'll revert it.

Reply:
Hack all the things 👍